include/dict0types.h

```diff
@@ -530,6 +534,50 @@ struct dict_table_t {
   IF_DEBUG(ulint magic_n;)
 };
 
+class dict_table_lru {
+ private:
+  using table_list_t = std::list<dict_table_t *>;
+  using list_itr_t = table_list_t::iterator;
+
+  struct cache_entry {
```

Comment:
For classes we need to move away from the current naming and use upper case first letter and snake case.